pool/cache.go

```diff
@@ -61,3 +68,15 @@ func (c *SessionCache) DeleteByPrefix(prefix string) {
 		}
 	}
 }
+
+func (c *SessionCache) UpdateEpoch(newEpoch uint64) {
```

Comment:
Unused in this commit, should it be used from elsewhere?

Reply:
This is used in https://github.com/nspcc-dev/neofs-sdk-go/pull/141/commits/56aa8e159cfc6c6ffe0a8fa9e2db41cc7996eb83